pages/swap.tsx

```diff
@@ -23,50 +23,19 @@ export async function getStaticProps() {
 	};
 }
 
-const Swap: React.FC<{ supportedAssets: CENNZAsset[] }> = ({
-	supportedAssets,
-}) => {
+const Swap: VFC<{ supportedAssets: CENNZAsset[] }> = ({ supportedAssets }) => {
```

Comment:
can you explain the difference between FC & VFC? reading up on it I thought VFC was better for a component that doesn't have any props

Reply:
Both them are almost identical, only difference is `VFC` implies the component doesn't accept `children` props, whereas `FC` does.

So if our component doesn't plan to use `children` props passing to it, then we should use `VFC` explicitly so Typescript can pick up and throw build error 

More details here https://www.mydatahack.com/using-react-vfc-instead-of-react-fc/, and here https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/